### ISSUE TITLE
`rubocop -a` should state how many offenses would be correctable with `-A`

### ISSUE THREAD

**Issue by marcandre** [rubocop-hq/rubocop#8362]
Also, `rubocop` should state how many of the offenses are auto-correctable.

**Comment by michaelnwani** [comment]
Relatedly, is -a no longer documented? Not sure if it was previously (am on v0.87.1)

```bash
Usage: rubocop [options] [file1, file2, ...]
    -L, --list-target-files          List all files RuboCop will inspect.
        --except [COP1,COP2,...]     Disable the given cop(s).
        --only [COP1,COP2,...]       Run only the given cop(s).
        --only-guide-cops            Run only cops for rules that link to a
                                     style guide.
    -c, --config FILE                Specify configuration file.
        --force-exclusion            Force excluding files specified in the
                                     configuration `Exclude` even if they are
                                     explicitly passed as arguments.
        --only-recognized-file-types Inspect files given on the command line only if
                                     they are listed in AllCops/Include parameters
                                     of user configuration or default configuration.
        --ignore-parent-exclusion    Prevent from inheriting AllCops/Exclude from
                                     parent folders.
        --force-default-config       Use default configuration even if configuration
                                     files are present in the directory tree.
        --auto-gen-config            Generate a configuration file acting as a
                                     TODO list.
        --exclude-limit COUNT        Used together with --auto-gen-config to
                                     set the limit for how many Exclude
                                     properties to generate. Default is 15.
        --disable-uncorrectable      Used with --auto-correct to annotate any
                                     offenses that do not support autocorrect
                                     with `rubocop:todo` comments.
        --no-offense-counts          Do not include offense counts in configuration
                                     file generated by --auto-gen-config.
        --auto-gen-only-exclude      Generate only Exclude parameters and not Max
                                     when running --auto-gen-config, except if the
                                     number of files with offenses is bigger than
                                     exclude-limit.
        --no-auto-gen-timestamp      Do not include the date and time when
                                     the --auto-gen-config was run in the file it
                                     generates.
        --init                       Generate a .rubocop.yml file in the current directory.
    -f, --format FORMATTER           Choose an output formatter. This option
                                     can be specified multiple times to enable
                                     multiple formatters at the same time.
                                     [p]rogress is used by default
                                       [a]utogenconf
                                       [c]lang
                                       [e]macs
                                       [fi]les
                                       [fu]ubar
                                       [h]tml
                                       [j]son
                                       [ju]nit
                                       [o]ffenses
                                       [pa]cman
                                       [p]rogress
                                       [q]uiet
                                       [s]imple
                                       [t]ap
                                       [w]orst
                                       custom formatter class name
    -o, --out FILE                   Write output to a file instead of STDOUT.
                                     This option applies to the previously
                                     specified --format, or the default format
                                     if no format is specified.
        --display-only-failed        Only output offense messages. Omit passing
                                     cops. Only valid for --format junit.
    -r, --require FILE               Require Ruby file.
        --fail-level SEVERITY        Minimum severity (A/R/C/W/E/F) for exit
                                     with error code.
        --display-only-fail-level-offenses
                                     Only output offense messages at
                                     the specified --fail-level or above
        --show-cops [COP1,COP2,...]  Shows the given cops, or all cops by
                                     default, and their configurations for the
                                     current directory.
    -F, --fail-fast                  Inspect files in order of modification
                                     time and stop after the first file
                                     containing offenses.
    -C, --cache FLAG                 Use result caching (FLAG=true) or don't
                                     (FLAG=false), default determined by
                                     configuration parameter AllCops: UseCache.
    -d, --debug                      Display debug info.
    -D, --[no-]display-cop-names     Display cop names in offense messages.
                                     Default is true.
    -E, --extra-details              Display extra details in offense messages.
    -S, --display-style-guide        Display style guide URLs in offense messages.
    -a, --auto-correct               Auto-correct offenses (only when it's safe).
        --safe-auto-correct          (same, deprecated)
    -A, --auto-correct-all           Auto-correct offenses (safe and unsafe)
        --disable-pending-cops       Run without pending cops.
        --enable-pending-cops        Run with pending cops.
        --ignore-disable-comments    Run cops even when they are disabled locally
                                     with a comment.
        --safe                       Run only safe cops.
        --[no-]color                 Force color output on or off.
    -v, --version                    Display version.
    -V, --verbose-version            Display verbose version.
    -P, --parallel                   Use available CPUs to execute inspection in
                                     parallel.
    -l, --lint                       Run only lint cops.
    -x, --fix-layout                 Run only layout cops, with auto-correct on.
    -s, --stdin FILE                 Pipe source from STDIN, using FILE in offense
                                     reports. This is useful for editor integration.
```
